README.md

```diff
@@ -202,6 +202,12 @@ You might also consider generating data with SparkSQL and then querying this dat
 
 1. Login Spark container and execute the SQLs:
 
+Before running the spark-shells, you need to set the `HADOOP_USER_NAME` environment variable to `anonymous`.
+
+```shell
+export HADOOP_USER_NAME=anonymous
```

Comment:
is there any way to add to spark configurations by default?

Reply:
Maybe I can add it to spark-env.sh.

Reply:
ok

Reply:
ok